enterprise/frontend/src/embedding-sdk/cli/steps/get-code-sample.ts

```diff
@@ -0,0 +1,54 @@
+import clipboard from "clipboardy";
+import toggle from "inquirer-toggle";
+
+import { getCodeSample } from "embedding-sdk/cli/constants/code-sample";
+import type { CliStepMethod } from "embedding-sdk/cli/types/types";
+import {
+  printEmptyLines,
+  printInfo,
+  printSuccess,
+} from "embedding-sdk/cli/utils/print";
+
+export const generateCodeSample: CliStepMethod = async state => {
+  if (!state.instanceUrl || !state.apiKey) {
+    return [
+      {
+        type: "error",
+        message: "Missing instance URL or API key",
+      },
+      state,
+    ];
+  }
+
+  printEmptyLines(2);
+  printSuccess(
+    "API key generated successfully. Here's a code sample to embed the Metabase SDK in your React application:",
+  );
+
+  const codeSample = getCodeSample(state.instanceUrl, state.apiKey);
+  printEmptyLines();
+  printInfo(codeSample.trim());
+  printEmptyLines();
+
+  const shouldCopyToClipboard = await toggle({
+    message: "Would you like to copy the code to your clipboard?",
+    default: true,
+  });
+
+  if (shouldCopyToClipboard) {
```

Comment:
@heypoom i added the `clipboard` package to make copying the code easier (i.e. just clicking 'yes' instead of the user having to copy things themselves). small qol upgrade, but I feel like little things like this will make users happy :smile:

Reply:
@oisincoveney This is a nice touch - thank you!